src/components/modal-component/shared-modal.css

```diff
@@ -0,0 +1,58 @@
+.shared-modal {
+    background-color: rgba(0, 0, 0, 0.5);
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+.modal-header {
+    opacity: 1;
```

Comment:
I think opacity, align-content and text-align are not making any impact on the modal header. Please. check.